controller/cache/cache.go

```diff
@@ -462,9 +469,10 @@ func (c *liveStateCache) getCluster(server string) (clustercache.ClusterCache, e
 			if isRoot && appName != "" {
 				res.AppName = appName
 			}
+
 			gvk := un.GroupVersionKind()
 
-			if shouldHashManifest(appName, gvk) {
+			if c.cacheSettings.ignoreResourceUpdatesEnabled && shouldHashManifest(appName, gvk) {
```

Comment:
Use local value. access to cache settings must be locked.

```suggestion
			if cacheSettings.ignoreResourceUpdatesEnabled && shouldHashManifest(appName, gvk) {
```